addons/account/migrations/11.0.1.1/post-migration.py

```diff
@@ -44,6 +44,10 @@ def migrate_account_tax_cash_basis(env):
             WHERE company_id = rc.id AND tax_exigibility = 'on_payment')""")
 
 
+def migrate_account_report_invoice_with_payments(env):
+    env.ref("account.report_invoice_with_payments").attachment_use = False
```

Comment:
I think this is not the report that is changing. Please check the correct XML-ID.

Reply:
I was using the report name, not the id. I think now is correct.